### DISCUSSION TITLE
Autocue Documentation, Tips, Help, Releases — Where to find them?

### DISCUSSION THREAD

**Discussion by Moonbase59** [AzuraCast/AzuraCast#7301] [General]
I am very happy about the great collaboration with BusterNeece, and that _Autocue_ is now also integrated with _AzuraCast_. Every station owner can now use _Autocue_ much more easily—it’s just a few button clicks and settings away, and the user experience much better.

But not everyone seems to be aware that _Autocue_ is an [independent project](https://github.com/Moonbase59/autocue), and thus its _documentation_ and _FAQ_ are kept in another repository.

I lately see _the same questions coming over and over again_, so here are the most **important links to help you along:**

- **Introduction to Autocue:** Updated [Liquidshop 4 Autocue Presentation](https://moonbase59.github.io/autocue/presentation/autocue.html)
- **Installation and Setup:** [README](https://github.com/Moonbase59/autocue/blob/master/README.md)
- **Help, Tips, Solutions:** [FAQ (Frequently Asked Questions)](https://github.com/Moonbase59/autocue/blob/master/FAQ.md)
- **What has changed?** [CHANGELOG](https://github.com/Moonbase59/autocue/blob/master/CHANGELOG.md)
- **Releases Overview:** [Releases](https://github.com/Moonbase59/autocue/releases) (latest first)
- **Sponsor my (Moonbase59’s) work:** [Donate](https://www.paypal.com/donate/?hosted_button_id=PBPR63362LDEU) — Thank you!

I hope this helps you along getting your station even _better_ — with _AzuraCast_ and _Autocue_ teamed up!

**Comment by gAlleb** [comment]
❤️
